venom/rpc/reflect/reflect.py

```diff
@@ -1,17 +1,26 @@
 from typing import Type, Set
 
 from venom import Message
-from venom.rpc import Method
+from venom.rpc.method import Method
 from venom.rpc import Service
 
 
 class Reflect(object):
-    services: Set[Service]
-    methods = Set[Method]
-    messages: Set[Type[Message]]
+    services = Set[Type[Service]]
+    methods = Set[Type[Method]]
+    messages = Set[Type[Message]]
 
     def __init__(self):
         self.services = set()
+        self.methods = set()
+        self.messages = set()
 
-    def add(self, service: Type[Service]):
+    def _add_method(self, method: Method):
+        self.messages.add(method.request)
```

Comment:
You probably also want to handle inspecting the message to look for fields that are messages here.